src/camera.py

```diff
@@ -0,0 +1,88 @@
+from os import getcwd
+
+from picar.SunFounder_PCA9685 import Servo
+from picar import filedb
+
+
+'''
+Camera class to control panning and tilting.
+
+Creates camera object that can be operated on using set_pan and set_tilt.
+set_pan operates [0,180] degrees. set_tilt operates [0,150] degrees. The 
+current values for pan and tilt in degrees can be returned using the get_pan
+and get_tilt functions. If the offset needs to be changed it is located in 
+"/../SunFounder_PiCar-V/remote_control/remote_control/driver/config"under
+pan_offset and tilt_offset. 
+
+'''
+class Camera:
+    PAN_CHANNEL = 1
+    TILT_CHANNEL = 2
+    MAX_PAN_DEGREE = 180
+    MAX_TILT_DEGREE = 150
+    MIN_DEGREE = 0
```

Comment:
I thought the camera's max and min angles for panning were 45 and 135 degrees?

Reply:
That's for turning